tests/backend/test_apiv1_organization_stats.py

```diff
@@ -70,3 +96,152 @@ async def test_organization_stats(
 async def test_stats_unknown_organization(administration_backend_sock):
     rep = await organization_stats(administration_backend_sock, organization_id="dummy")
     assert rep == {"status": "not_found"}
+
+
+@pytest.fixture
+async def access_testbed(
+    backend_factory,
+    backend_data_binder_factory,
+    apiv1_backend_sock_factory,
+    organization_factory,
+    local_device_factory,
+):
+
+    async with backend_factory(populated=False) as backend:
+        binder = backend_data_binder_factory(backend)
+
+        org = organization_factory("IFD")
+        device = local_device_factory(
+            org=org,
+            base_device_id="godfrey@d1",
+            base_human_handle="Godfrey Ho <godfrey.ho@ifd.com>",
+            profile=UserProfile.ADMIN,
+        )
+        with freeze_time("2000-01-01"):
+            await binder.bind_organization(org, device)
+        async with apiv1_backend_sock_factory(backend, "administration") as sock:
+
+            yield binder, org, device, sock
+
+
+@pytest.mark.trio
+async def test_organization_stats_users(access_testbed, core_factory, local_device_factory):
+    binder, org, godfrey1, sock = access_testbed
+    stats = await organization_stats(sock, org.organization_id)
+    assert stats == {
+        "status": "ok",
+        "users": 1,
+        "active_users": 1,
+        "users_per_profile_detail": [
+            {"profile": UserProfile.ADMIN, "active": 1, "revoked": 0},
+            {"profile": UserProfile.STANDARD, "active": 0, "revoked": 0},
+            {"profile": UserProfile.OUTSIDER, "active": 0, "revoked": 0},
+        ],
+        "data_size": 0,
+        "metadata_size": ANY,
+        "workspaces": 1,
+    }
+
+    nick1 = local_device_factory(
+        base_device_id="el_murcielago_enmascarado_ii@d1",
+        base_human_handle="Rodolfo Guzman Huerta",
+        profile=UserProfile.ADMIN,
+        org=org,
+    )
+    await binder.bind_device(nick1, certifier=godfrey1)
```

Comment:
It might be more readable to use a loop here:
```python
expected_stats = {
        "status": "ok",
        "users": 1,
        "active_users": 1,
        "users_per_profile_detail": [
            {"profile": UserProfile.ADMIN, "active": 1, "revoked": 0},
            {"profile": UserProfile.STANDARD, "active": 0, "revoked": 0},
            {"profile": UserProfile.OUTSIDER, "active": 0, "revoked": 0},
        ],
        "data_size": 0,
        "metadata_size": ANY,
        "workspaces": 2,
    }

for profile in UserProfile:
    device = local_device_factory(profile=profile, org=org)
    await binder.bind_device(device, certifier=godfrey1, initial_user_manifest_in_v0=True)
    expected_stats["users"] += 1
    expected_stats["active_users"] += 1
    expected_stats["users_per_profile_detail"][profile]["active"] += 1
    stats = await organization_stats(sock, org.organization_id)
    assert stats == expected_stats

    await binder.bind_revocation(device.user_id, certifier=godfrey1)
    expected_stats["active_users"] -= 1
    expected_stats["users_per_profile_detail"][profile]["active"] -= 1
    expected_stats["users_per_profile_detail"][profile]["revoked"] += 1
    stats = await organization_stats(sock, org.organization_id)
    assert stats == expected_stats

# Also make sure stats are isolated between organizations
device = local_device_factory(org=otherorg, profile=UserProfile.ADMIN)
await binder.bind_organization(otherorg, device, initial_user_manifest_in_v0=True)
stats = await organization_stats(sock, org.organization_id)
assert stats == expected_stats
```